documentation/examples/prometheus-kubernetes.yml

```diff
@@ -74,19 +68,53 @@ scrape_configs:
     replacement: $1:$2
   - action: labelmap
     regex: __meta_kubernetes_service_label_(.+)
-    replacement: $1
   - source_labels: [__meta_kubernetes_role]
     action: replace
-    regex: (.+)
     target_label: kubernetes_role
-    replacement: $1
   - source_labels: [__meta_kubernetes_service_namespace]
     action: replace
-    regex: (.+)
     target_label: kubernetes_namespace
-    replacement: $1
   - source_labels: [__meta_kubernetes_service_name]
```

Comment:
No big objection, other than to just keep Kubernetes meta labels grouped together with `kubernetes_` prefix. But if you think `job` makes sense then sure.